### PR TITLE
[feature] Add k8s clean configmap

### PR DESCRIPTION
**Pull request by ernelio**
**- What I did**
- rit k8s clean helm-configmap

### CODE REVIEW

File: kubernetes/clean/helm-configmap/config.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "description": "Helm clean configmap for Ritchie."
```
Comment (viniciusramosdefaria) [inline kubernetes/clean/helm-configmap/config.json:2]: helm configmap cleaner

File: tree/tree.json [hunk 2]
```diff
@@ -310,6 +310,26 @@
...
+    },
+    {
+      "usage": "helm-configmap",
+      "help": "clean helm-configmap",
+      "formula": {
+        "path": "kubernetes/clean/helm-configmap",
+        "bin": "helm-configmap.sh",
+        "binLinux": "helm-configmap.sh",
+        "binDarwin": "helm-configmap.sh",
+        "binWindows": "helm-configmap.bat",
```
Comment (viniciusramosdefaria) [inline tree/tree.json:326]: there isn't a bat for it

File: kubernetes/clean/helm-configmap/config.json (hunk 1 above)
Comment (erneliojuniorzup) [reply kubernetes/clean/helm-configmap/config.json:2]: ok

File: tree/tree.json (hunk 2 above)
Comment (erneliojuniorzup) [reply tree/tree.json:326]: done

**Pull request merged**